tests/playbooks/test_minimal.yaml

```diff
@@ -39,6 +39,9 @@
     - role: swift_adoption
       tags:
         - swift_adoption
+    - role: cinder_adoption
```

Comment:
Please make a follow up PR to make the same ordering switch in the documentation too. Not blocking this PR.

Reply:
Ack